common/src/main/java/com/gentics/mesh/dagger/MeshComponent.java

```diff
@@ -64,6 +62,14 @@
 		 */
 		Builder searchProviderType(@Nullable SearchProviderType type);
 
+		/**
+		 * Inject the own instance of {@link SearchWaitUtil}.
+		 * 
+		 * @param type
+		 * @return
+		 */
+		Builder searchWaitUtilSupplier(@Nullable Supplier<SearchWaitUtil> swUtil);
```

Comment:
The original implementation of `SearchWaitUtil` overriding used a special set of Dagger modules, set in OrientDBMeshComponent, and overridden in MeshTestComponent. Since MeshTestComponent cannot extend OrientDBMeshComponent anymore, a new module overriding should be introduced. Here we allow a selected module (SearchWaitUtil) to be overridden with an optional supplier.